package.json

```diff
@@ -60,9 +60,9 @@
   "dependencies": {
     "@babel/types": "^7.16.0",
     "@randlabs/myalgo-connect": "^1.1.1",
-    "@reach-sh/stdlib": "^0.1.10-rc.3",
+    "@reach-sh/stdlib": "0.1.11-rc.0",
```

Comment:
Let's make sure we release with the most up to date version of reach. This might mean we need to recompile & update the contracts package